From 826fca51b6d13696f7c05c44e3d79ce459be3cf6 Mon Sep 17 00:00:00 2001
From: ionian <mike.github778@gmail.com>
Date: Sun, 3 Feb 2019 03:04:46 -0800
Subject: [PATCH] made base config changes

---
 config.def.h | 31 ++++++++++++++++++++++---------
 1 file changed, 22 insertions(+), 9 deletions(-)

diff --git a/config.def.h b/config.def.h
index 77ff358..cf7def2 100644
--- a/config.def.h
+++ b/config.def.h
@@ -9,28 +9,34 @@ static const char selbordercolor[]  = "#005577";
 static const char selbgcolor[]      = "#005577";
 static const char selfgcolor[]      = "#eeeeee";
 static const unsigned int borderpx  = 1;        /* border pixel of windows */
-static const unsigned int snap      = 32;       /* snap pixel */
+static const unsigned int snap      = 8;        /* snap pixel */
 static const Bool showbar           = True;     /* False means no bar */
 static const Bool topbar            = True;     /* False means bottom bar */
 
 /* tagging */
-static const char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
+static const char *tags[] = { "base", "web", "term", "mail", };
 
 static const Rule rules[] = {
-	/* class      instance    title       tags mask     isfloating   monitor */
-	{ "Gimp",     NULL,       NULL,       0,            True,        -1 },
-	{ "Firefox",  NULL,       NULL,       1 << 8,       False,       -1 },
+   /* class      instance    title       tags mask     isfloating   monitor */
+   { "Vimprobable", NULL,       NULL,       1 << 1,       False,       -1 },
+   { "chromium",    NULL,       NULL,       1 << 2,       False,       -1 },
+   { "tabbed",      NULL,       NULL,       1 << 1,       False,       -1 },
+   { "mpv",         NULL,       NULL,       1 << 0,       True,        -1 },
+   {  NULL,         NULL,      "mutt",      1 << 3,       False,       -1 },
+   {  NULL,         NULL,      "tmux",      1 << 2,       False,       -1 },
+   {  NULL,         NULL,      "ssh",       1 << 0,       False,       -1 },
+   {  NULL,         NULL,      "scratchpad",     0,       True,        -1 },
 };
 
 /* layout(s) */
 static const float mfact      = 0.55; /* factor of master area size [0.05..0.95] */
 static const int nmaster      = 1;    /* number of clients in master area */
-static const Bool resizehints = True; /* True means respect size hints in tiled resizals */
+static const Bool resizehints = False;/* True means respect size hints in tiled resizals */
 
 static const Layout layouts[] = {
 	/* symbol     arrange function */
-	{ "[]=",      tile },    /* first entry is default */
-	{ "><>",      NULL },    /* no layout function means floating behavior */
+	{ "[T]",      tile },    /* first entry is default */
+	{ "[F]",      NULL },    /* no layout function means floating behavior */
 	{ "[M]",      monocle },
 };
 
@@ -47,7 +53,14 @@ static const Layout layouts[] = {
 
 /* commands */
 static const char *dmenucmd[] = { "dmenu_run", "-fn", font, "-nb", normbgcolor, "-nf", normfgcolor, "-sb", selbgcolor, "-sf", selfgcolor, NULL };
-static const char *termcmd[]  = { "uxterm", NULL };
+static const char   *termcmd[] = { "urxvtc", NULL };
+static const char  *xtermcmd[] = { "xterm",  NULL };
+static const char   *mailcmd[] = { "urxvtc", "-title", "mutt", "-e", "mutt", NULL };
+static const char   *tmuxcmd[] = { "urxvtc", "-title", "tmux", "-e", "tmux", "-f", "/home/jason/.tmux/conf", NULL };
+static const char    *padcmd[] = { "urxvtc", "-title", "scratchpad", "-geometry", "56x10-30+40", NULL };
+static const char   *lockcmd[] = { "xautolock", "-locknow", NULL };
+static const char *rebootcmd[] = { "systemctl", "reboot", NULL };
+static const char   *shutcmd[] = { "systemctl", "poweroff", NULL };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
-- 
2.20.1

